source/documentation/guidance/pentest.md

```diff
@@ -26,12 +23,7 @@ Send the below information to [gov-uk-paas-support@digital.cabinet-office.gov.uk
 
 You should supply as much information as possible to minimise delays in this process.
 
-AWS responds to permission requests within two business days. The PaaS team will forward the AWS response to you within two further business days of receiving it.
-
-If your request is approved, you will receive an authorisation number. Your penetration test is only authorised for the dates and times you have specified.
-
-If your request is not approved, the PaaS team will work with you and AWS to solve any issues. 
```

Comment:
What happens if your request is not approved?

Reply:
The answer to this was we would probably never reject a request. In which case, are you happy that wording below is sufficient, @jonathanglassman?

> so that we are aware of [your pen test request] and can approve it.

Reply:
Sorry, forgot to reply to this! Yes that sounds good, can we say "penetration" rather than "pen" i.e. `so that we are aware of your penetration test request and can approve it.`